src/web/mjs/engine/Manga.mjs

```diff
@@ -22,7 +22,7 @@ export default class Manga {
         this.status = status;
         this.chapterCache = [];
         this.existingChapters = [];
-
+        this.hasUpdate=false;
```

Comment:
Maybe use more expressive name as already done in the UI, e.g. `hasNewContent`, `hasNewChapters`, ...